src/components/DepositWidget/Row.tsx

```diff
@@ -40,7 +58,22 @@ function _loadFallbackTokenImage(event: React.FormEvent<HTMLImageElement>): void
   image.src = unknownTokenImg
 }
 
-export const Row: React.FC<RowProps> = ({ tokenBalances }: RowProps) => {
+const txOptionalParams: TxOptionalParams = {
+  onSentTransaction: (receipt: Receipt): void => {
+    toast.info(
+      <div>
```

Comment:
This much code looks like it could be its own component
`<TxPopup hash={}/>` maybe?

Reply:
Sure. I can do that in another PR.